itk-module.cmake

```diff
@@ -1,31 +1,37 @@
-set(DOCUMENTATION "This module contains a class to perform automatic skull-stripping for neuroimage analysis.
-It is based on the ITK level-set and registration frameworks." )
+# the top-level README is used for describing this module, just
+# re-used it for documentation here
+get_filename_component(MY_CURENT_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+file(READ "${MY_CURENT_DIR}/README.rst" DOCUMENTATION)
 
-itk_module( SkullStrip
+# itk_module() defines the module dependencies in SkullStrip
+# The testing module in SkullStrip depends on ITKTestKernel
+# By convention those modules outside of ITK are not prefixed with
+# ITK
+
+# define the dependencies of the include module and the tests
+itk_module(SkullStrip
   DEPENDS
     ITKLevelSets
     ITKRegistrationCommon
     ITKIOImageBase
     ITKImageIntensity
     ITKIOMeta
-# Image IO Modules
-  ITKIOJPEG
-  ITKIOGDCM
-  ITKIOBMP
-  ITKIOLSM
-  ITKIOPNG
-  ITKIOTIFF
-  ITKIOVTK
-  ITKIOStimulate
-  ITKIOBioRad
-  ITKIOMeta
-  ITKIONIFTI
-  ITKIONRRD
-  ITKIOGIPL
-# Transform IO Modules
-  ITKIOTransformMatlab
-  ITKIOTransformHDF5
-  ITKIOTransformInsightLegacy
+    ITKIOJPEG
+    ITKIOGDCM
+    ITKIOBMP
+    ITKIOLSM
+    ITKIOPNG
+    ITKIOTIFF
+    ITKIOVTK
+    ITKIOStimulate
+    ITKIOBioRad
+    ITKIOMeta
+    ITKIONIFTI
+    ITKIONRRD
+    ITKIOGIPL
+    ITKIOTransformMatlab
+    ITKIOTransformHDF5
+    ITKIOTransformInsightLegacy
```

Comment:
Not sure whether all these need to be explicitly listed. If anyone happens to know that they can be called as a group using another module name, please comment.

Reply:
`ITKImageIO` and `ITKTransformIO`. But I am not sure we can use them in `itk-module.cmake`. Link: https://discourse.itk.org/t/find-all-available-modules-from-groupio/351/12?u=dzenanz